gp-templates/translation-row-editor.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Template for the editor part of a single translation row in a translation set display
+ *
+ * @package    GlotPress
+ * @subpackage Templates
+ */
+
+$colspan = apply_filters( 'gp_translation_row_editor_colspan', $can_approve ? 5 : 4 );
+
+$singular = sprintf(
+	/* translators: %s: Original singular form of the text */
+	__( 'Singular: %s', 'glotpress' ),
+	'<span class="original">' . ( isset( $t->singular_glossary_markup ) ? $t->singular_glossary_markup : esc_translation( $t->singular ) ) . '</span>'
+);
+$plural = sprintf(
+	/* translators: %s: Original plural form of the text */
+	__( 'Plural: %s', 'glotpress' ),
+	'<span class="original">' . ( isset( $t->plural_glossary_markup ) ? $t->plural_glossary_markup : esc_translation( $t->plural ) ) . '</span>'
+);
+
+?>
+<tr class="editor <?php gp_translation_row_classes( $t ); ?>" id="editor-<?php echo esc_attr( $t->row_id ); ?>" row="<?php echo esc_attr( $t->row_id ); ?>">
+	<td colspan="<?php echo esc_attr( $colspan ); ?>">
+		<div class="strings">
+			<?php if ( ! $t->plural ) : ?>
+				<p class="original"><?php echo prepare_original( $singular );  // WPCS: XSS OK. ?></p>
+				<?php textareas( $t, array( $can_edit, $can_approve_translation ) ); ?>
+			<?php else : ?>
+				<?php if ( absint( $locale->nplurals ) === 2 && 'n != 1' === $locale->plural_expression ) : ?>
+					<p>
+						<?php echo $singular;  // WPCS: XSS OK. ?>
+					</p>
+					<?php textareas( $t, array( $can_edit, $can_approve ), 0 ); ?>
+					<p class="clear">
+						<?php echo $plural;  // WPCS: XSS OK. ?>
+					</p>
+					<?php textareas( $t, array( $can_edit, $can_approve ), 1 ); ?>
+				<?php else : ?>
+					<!--
+					TODO: labels for each plural textarea and a sample number
+					-->
+					<p>
+						<?php echo $singular;  // WPCS: XSS OK. ?>
+					</p>
+					<p class="clear">
+						<?php echo $plural;  // WPCS: XSS OK. ?>
+					</p>
+					<?php foreach ( range( 0, $locale->nplurals - 1 ) as $plural_index ) : ?>
+						<?php if ( $locale->nplurals > 1 ) : ?>
+							<p class="plural-numbers">
+								<?php
+								printf(
+									/* translators: %s: Numbers */
+									__( 'This plural form is used for numbers like: %s', 'glotpress' ),
+									'<span class="numbers">' . implode( ', ', $locale->numbers_for_index( $plural_index ) ) . '</span>'
+								); // WPCS: XSS OK.
+								?>
+							</p>
+						<?php endif; ?>
+						<?php textareas( $t, array( $can_edit, $can_approve ), $plural_index ); ?>
+					<?php endforeach; ?>
+				<?php endif; ?>
+			<?php endif; ?>
+			<?php gp_tmpl_load( 'translation-row-editor-actions', get_defined_vars() ); ?>
+		</div>
+		<?php gp_tmpl_load( 'translation-row-editor-meta', get_defined_vars() ); ?>
+	</td>
+	<?php
+	/**
+	 * Fires after editor column
+	 *
+	 * @since 3.0.0
+	 *
+	 * @param GP_Translation $t The current translation.
+	 * @param GP_Translation_Set $translation_set The current translation set.
```

Comment:
The variables should be aligned.

Reply:
(Wondering if should take the opportunity to rename `$t` to `$translation` for clarity.)

Reply:
Why not. I'm a fan of descriptive variable names.